src/main/java/wooteco/subway/admin/service/PathService.java

```diff
@@ -0,0 +1,137 @@
+package wooteco.subway.admin.service;
+
+import org.springframework.stereotype.Service;
+import wooteco.subway.admin.domain.Line;
+import wooteco.subway.admin.domain.LineStation;
+import wooteco.subway.admin.domain.PathType;
+import wooteco.subway.admin.domain.Station;
+import wooteco.subway.admin.dto.PathResponse;
+import wooteco.subway.admin.dto.StationResponse;
+import wooteco.subway.admin.repository.LineRepository;
+import wooteco.subway.admin.repository.StationRepository;
+import wooteco.subway.admin.service.errors.PathException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Service
+public class PathService {
+    private static final String KOREAN_WORD = "^[가-힣]*$";
+
+    private StationRepository stationRepository;
```

Comment:
```suggestion
    private final StationRepository stationRepository;
```
final을 붙여서 필요한 클래스를 명확하게 표시해주는게 좋습니다